kANe/v1p1/03_date_of_dharmashAstra_works.md

```diff
@@ -1,173 +1,100 @@
 +++
-title = "03 Date of Dharmaśāstra works"
-short_title = "03 Date of Dharmaśāstra"
+title = "03 When Dharmasastra works were first composed"
```

Comment:
namaste suhAs - please revert changes to title and short-title lines. They're being deliberately kept short. The full title should be presented in line 6 instead.